File: multi-gammon-core/src/main/java/org/akoshterek/backgammon/matchstate/MatchMove.java
package org.akoshterek.backgammon.matchstate;

import org.akoshterek.backgammon.move.MoveRecord;

import java.util.Deque;
import java.util.LinkedList;

/**
 * @author Alex
 *         date 06.08.2015.
 */
public class MatchMove {
    public final Deque<MoveRecord> moveRecords = new LinkedList<>();
}

File: multi-gammon-core/src/main/java/org/akoshterek/backgammon/matchstate/MatchMove.scala
package org.akoshterek.backgammon.matchstate

import org.akoshterek.backgammon.move.MoveRecord

import scala.collection.mutable

/**
  * @author Alex
  *         date 06.08.2015.
  */
final class MatchMove {
  val moveRecords: mutable.Buffer[MoveRecord] = mutable.Buffer[MoveRecord]()
}